CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.11.0] - 2024-01-03
+### New
```

Comment:
Not critical, but if we're following [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), this should be `Added` instead of `New`.